src/server/storage/storageAPI.js

```diff
@@ -29,13 +30,34 @@ const setup = (app: Router, storage: StorageAPI) => {
       const { body, user: userRecord } = req
       const logger = req.log
       logger.debug('new user request:', { data: body.user, userRecord })
+
+      //if torus, then we first verify the user mobile/email by verifying it matches the torus public key
+      //(torus maps identifier such as email and mobile to private/public key pairs)
+      if (body.user.torusProof) {
```

Comment:
we could use desctructuring assignment for better look & shorter code

Reply:
done